src/App.tsx

```diff
@@ -1,24 +1,163 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import cn from 'classnames';
+import { useEffect, useState, useCallback } from 'react';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { FilteredTodo } from './components/FilteredTodo';
+import { Filter } from './types/Filter';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { TodoForm } from './components/CreateTodoForm';
 
-const USER_ID = 0;
+const USER_ID = 10400;
 
 export const App: React.FC = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.ALL);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTodo, setNewTodo] = useState<Todo | null>(null);
+  const [isTodoCreate, setIsTodoCreate] = useState(false);
+
+  const handleTodoCreate = useCallback(async (title: string) => {
+    if (!title) {
+      setError('Title is empty, Why bro?????');
+
+      return;
+    }
+
+    setIsTodoCreate(true);
+
+    const todo = {
+      userId: USER_ID,
+      title,
+      completed: false,
+    };
+
+    try {
+      const createdTodo = await createTodo(todo);
+
+      setTodos((prevTodos) => [...prevTodos, createdTodo]);
+    } catch ({ message }) {
+      setError(`Error ${message}`);
+    } finally {
+      setIsTodoCreate(false);
+      setNewTodo(null);
+    }
+  }, []);
+
+  const handleDelete = useCallback(async () => {
+    const completedTodoIds = todos
+      .filter((todo) => todo.completed)
+      .map((todo) => todo.id);
+
+    try {
+      await Promise.all(completedTodoIds.map((todoId) => deleteTodo(todoId)));
+
+      const updatedTodos = todos.filter((todo) => !todo.completed);
+
+      setTodos(updatedTodos);
+    } catch ({ message }) {
+      setError(`Error ${message}}`);
+    }
+  }, [todos]);
+
+  useEffect(() => {
+    getTodos(USER_ID)
```

Comment:
Choose one of the ways & use one
1. async / await / try / catch
2. then / catch ..